app/default/src/Elasticsearch/Service/ApiClient.php

```diff
@@ -11,12 +11,13 @@
 
 final class ApiClient implements ApiClientInterface
 {
-	private const ELASTICSEARCH_HOST = 'http://localhost:9200';
+	private $ELASTICSEARCH_HOST;
```

Comment:
@212223 let's make it camel cased for the sake of consistency across the project 😉 
```suggestion
	private $elasticsearchHost;
```

Reply:
You're right. Done in [29e9892](https://github.com/lrynek/painless-car-rental/pull/1/commits/29e98920e2670e342a85c5cd17634d02ee5a078a)